code/templates/playwithsort.cpp

```diff
@@ -3,13 +3,13 @@
 #include <string>
 #include <iostream>
 #include <algorithm>
+#include <ranges>
 
-/*
-struct OrderFunctor {
-    bool operator() (const TYPE &s, const TYPE &t) const {
+struct RevOrderString {
+    bool operator() (const std::string &s, const std::string &t) const {
+        return std::ranges::lexicographical_compare(std::views::reverse(s), std::views::reverse(t));
```

Comment:
Is this supposed to be here or only in the solution? If it's only supposed to be in the solution, I would nevertheless include some hints like "check the function std::ranges::lexicographical_compare for comparing strings.

Reply:
I would indeed put only a hint, so that people have to dig a bit cppreference. Having it fully empty as it was was probably a bit too hard though

Reply:
I thought I add the full implementation so people have

1. an example for how the comparison functor should look like
2. don't need to spend time to figure out how to implement the reverse string comparison and focus on the template stuff.
3. 
But it's not necessary, I can omit it.

Reply:
You have a point that the goal of the exercise is not to dig cppreference and that it's already hard enough for students without implementing that. So ok, let's keep the implementation